src/react-components/src/components/01-atoms/Text/Headings/Heading.tsx

```diff
@@ -17,9 +17,11 @@ export interface HeadingProps {
 }
 
 export default function Heading(props: React.PropsWithChildren<HeadingProps>) {
-  const { level, id, baseClass = "heading", modifiers, url, urlClass, text, blockName, attributes } = props;
+  const { level, id, baseClass, modifiers, url, urlClass, text, blockName, attributes } = props;
+
+  console.log("baseClass", baseClass);
+  let heading_base_class = baseClass ? baseClass : "heading";
```

Comment:
The default value didn't work?

Reply:
It was a different styles issue.  It does work!